src/config.ts

```diff
@@ -68,7 +73,7 @@ class Config {
 
   /**
    * Gets the configured api key for tx-mining-service
-   * 
```

Comment:
The linter wouldn't allow me to ignore this line's trailing spaces, so I let it fix it.